tests/layer_tests/pytorch_tests/test_select.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+import numpy as np
+import torch
+
+
+@pytest.mark.parametrize('input_dim', list(range(-3, 4)))
+@pytest.mark.parametrize('input_index', list(range(-3, 4)))
+class TestSelect(PytorchLayerTest):
+
+    def _prepare_input(self):
+        return (np.random.randn(4, 4, 5, 5).astype(np.float32),)
+
+    def create_model(self, input_dim, input_index):
+
+        class aten_select(torch.nn.Module):
+
+            def __init__(self, input_dim, input_index) -> None:
+                super().__init__()
+                self.dim = torch.from_numpy(
```

Comment:
You do not need to create torch tensor here, because in `forward` you use it just as int. Please remove that from `dim` and `index`. Other then that everything is good.

Reply:
Sure, good idea :) I have corrected it.